github/integration.go

```diff
@@ -28,11 +28,11 @@ func (s *IntegrationsService) ListInstallations(opt *ListOptions) ([]*Installati
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeIntegrationPreview)
 
-	i := new([]*Installation)
+	var i []*Installation
 	resp, err := s.client.Do(req, &i)
```

Comment:
I wonder if this even worked before, given the double pointer given to `client.Do`, i.e., `**[]*Installation`.

A quick test reveals that it should be fine, see https://play.golang.org/p/O5lq1RfseF. But it's still nice to fix to have simpler code.